src/amo/pages/Home/styles.scss

```diff
@@ -146,27 +146,3 @@
     }
   }
 }
-
-.Icon-Home-SubjectShelf-ad-blockers {
-  background-image: url('./img/ad-blockers.svg');
-}
-
-.Icon-Home-SubjectShelf-password-managers {
-  background-image: url('./img/password-managers.svg');
-}
-
-.Icon-Home-SubjectShelf-bookmark-managers {
-  background-image: url('./img/bookmark-managers.svg');
-}
-
-.Icon-Home-SubjectShelf-watching-videos {
-  background-image: url('./img/watching-videos.svg');
-}
-
-.Icon-Home-SubjectShelf-smarter-shopping {
-  background-image: url('./img/smarter-shopping.svg');
-}
-
-.Icon-Home-SubjectShelf-be-more-productive {
-  background-image: url('./img/be-more-productive.svg');
```

Comment:
Would it make sense to remove the `svg` files too?

Reply:
Good idea. 👍